main.cpp

```diff
@@ -45,6 +87,18 @@ struct Human
     bool isHungry(int timeAwake, int stamina);
 };
 
+bool Human::isHungry(int timeAwake, int stamina)
+{
+    if (timeAwake > 8 && stamina <4 ) 
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
```

Comment:
you can skip the `else { return false; }` and just `return false;`   If the `if()` expression is false, execution will skip over that block and go right to line 96.

- [ ] simplify.